mep3_simulation/webots_data/protos/ExcavationSquare.proto

```diff
@@ -0,0 +1,167 @@
+  field SFRotation rotation     0 1 0 0
+  field SFString{"yellow1", "yellow2", "purple1", "purple2", "red"}  frontTexture  "yellow1"
+]
+{
+Solid {
+  translation IS translation
+  rotation IS rotation
+  children [
+    Group {
+      children [
+        DEF excSquare_boundingObject Transform {
+          translation 0 -0.0075 0.035
+          children [
+            Shape {
+              appearance DEF WALL_APPEARANCE PBRAppearance {
+		baseColor 0.631 0.612 0.537
+		roughness 1
+		metalness 0
+	      }
+              geometry Box {
+                size 0.185 0.015 0.07
+              }
+            }
+          ]
+        }
+        Transform {
+          translation -0.085 -0.0425 0.0075
+          children [
+            DEF excSquare_bottom Shape {
+              appearance USE WALL_APPEARANCE
+              geometry Box {
+                size 0.015 0.055 0.015
+              }
+            }
+          ]
+        }
+        Transform {
+          translation 0.085 -0.0425 0.0075
+          children [
+            USE excSquare_bottom
+          ]
+        }
+        Transform {
+          translation -0.085 -0.02925 0.0425
+          children [
+            DEF excSquare_first Shape {
+              appearance USE WALL_APPEARANCE
+              geometry Box {
+                size 0.015 0.0285 0.055
+              }
+            }
+          ]
+        }
+        Transform {
+          translation 0.085 -0.02925 0.0425
+          children [
+            USE excSquare_first
+          ]
+        }
+        Transform {
+          translation -0.085 -0.06175 0.0425
+          children [
+            DEF excSquare_second Shape {
+              appearance USE WALL_APPEARANCE
+              geometry Box {
+                size 0.015 0.0165 0.055
+              }
+            }
+          ]
+        }
+        Transform {
+          translation 0.085 -0.06175 0.0425
+          children [
+            USE excSquare_second
+          ]
+        }
+      ]
+    }
+    HingeJoint {
+      jointParameters HingeJointParameters {
+        position -0.4
+        anchor 0 -0.0485 0.02
+        minStop -0.401426
+        maxStop 2.6
+      }
+      endPoint Solid {
+        translation 0 -0.0485 0.02
+        rotation -1 0 0 0.4
+        children [
+          Transform {
+            translation 0 0 0.064
+            children [
+              Shape {
+                appearance USE WALL_APPEARANCE
+                geometry DEF excSquare_half Box {
+                  size 0.15 0.021 0.15
+                }
+              }
+            ]
+          }
+          Transform {
+            translation 0 0.0105 0.064
+            children [
+              Shape {
+                appearance PBRAppearance {
+                  baseColorMap ImageTexture {
+                        url [ %<= '"assets/samples/excSq_' + fields.frontTexture.value + '.png"' >% ]
+                    repeatS FALSE
+                    repeatT FALSE
```

Comment:
```suggestion
                  baseColorMap ImageTexture {
                    url [ %<= '"assets/samples/excSq_' + fields.frontTexture.value + '.png"' >% ]
                    repeatS FALSE
                    repeatT FALSE
```
Nazalost nema formater za PROTO fajlove, pa moras runcno uvlaciti :/

Reply:
clang-format radi za protobuf fajlove, testirao sam ga. treba podesiti .clang-format da bi ga formatirao na čitljiv način